.github/workflows/profiling.yaml

```
@@ -0,0 +1,101 @@
+        virtualenvs-in-project: true
+        installer-parallel: true
+
+    - name: Install dependencies
+      run: |
+        poetry install --no-interaction --all-extras
+        poetry add scalene 
+
+
+    # Set environment variables for SHAs
+    - name: Set environment variables
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:62: Double quote to prevent globbing and word splitting

(shellcheck)